### PR TITLE
Update challenge description with more background and context

### PR DESCRIPTION
**Pull request by egrace479**
I incorporated @pujols' changes with some minor edits.

@DavidCarlyn, if you made a new copy of the `HDR_fig.png` with the images from the presentation, could you add that into the `pages` directory? Thanks!

We still need to update the dates and the terms page, but I'll do that after we get a CodaBench challenge up for red-teaming.

### CODE REVIEW

File: pages/data.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-This challenge uses data compiled from various Zenodo records of the Butterfly Genetics Group. [More Info Coming]
+This challenge uses data compiled from various Zenodo records of the Butterfly Genetics Group.
 
 ## Instructions to Download Training Data
 
 First, install the downloader in your virtual environment:
 ```bash
-pip install git+https://github.com/Imageomics/cautious-robot.git@v0.1.1-alpha
+pip install git+https://github.com/Imageomics/cautious-robot.git@v0.2.0-alpha
```
[the remark this answers is not in the document]
Comment (egrace479) [reply pages/data.md:9]: `sum-buddy` is still on alpha release; would it be reasonable to just change to install from the repo (no release)?

File: pages/evaluation.md [hunk 2]
```diff
@@ -1,35 +1,47 @@
...
 
-Moreover, Species A has a mimic (or co-mimic) Species B. These two butterfly species have quite similar appearances to evade predators. The test data thus further contains:
-- A mimic butterfly species' (denoted as Species B) subspecies. Specifically, we only consider the two subspecies that correspond to the parent subspecies of Species A's signal hybrid.
-- Hybrids of these two subspecies.
-- The aim of this test set is to see whether an anomaly detection algorithm for butterflies is transferrable across mimic butterflies.
+The goal of the challenge is to distinguish between non-hybrid instances and hybrid instances.
 
-There are 2 phases:
+## The Second Component of the Test Dataset
+Moreover, Species A has a mimic Species B. These two butterfly species have quite similar appearances to evade predators. Specifically, for the two subspecies of Species A that produce the signal hybrid, there are mimic subspecies of Species B which also hybridize. We thus provide a separate, second test data set that contains:
```
Comment (egrace479) [inline pages/evaluation.md:12]:
```suggestion
Moreover, Species A has a [mimic](https://en.wikipedia.org/wiki/M%C3%BCllerian_mimicry) Species B. These two butterfly species have quite similar appearances to evade predators. Specifically, for the two subspecies of Species A that produce the signal hybrid, there are mimic subspecies of Species B which also hybridize. We thus provide a separate, second test data set that contains:
```

File: pages/data.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hlapp) [reply pages/data.md:9]: Yes in principle, with the caveat being that then someone who runs this 3 months from now may get something different from someone who runs it now. But that could be intended – for example, if there are improvements and/or bug fixes on the main branch by then, we would probably want someone to get these rather than a version that at that point won't have them.

I guess if we have people install from the main branch, we essentially commit ourselves to keeping that branch backwards compatible and tested for the duration of this competition. If we don't want to make that commitment, we should designate perhaps a special branch, to which we would keep migrating fixes from the main branch for the duration of the competition.

[the remark this answers is not in the document]
Comment (hlapp) [reply pages/data.md:9]: And as an aside, that a dependency of a software is alpha by no means has to imply that the software itself becomes "tainted" as alpha then too. If your software has a dependency that is alpha but your software otherwise is not, you essentially take responsibility for having checked that it works to the extent you need it to work. (The dependency may be alpha for reasons that don't matter to your software.)

[the remark this answers is not in the document]
Comment (egrace479) [reply pages/data.md:9]: There is one more large-ish change we were discussing ([here](https://github.com/Imageomics/cautious-robot/issues/15#issuecomment-2273710520)). I think we have pretty good testing in place, so we could update and do a release. However, I think that just having the newest version would likely be best; I don't intend to change the input in a way that wouldn't be backwards compatible.

Comment (egrace479) [inline pages/data.md:9]:
```suggestion
pip install git+https://github.com/Imageomics/cautious-robot
```

File: pages/overview.md [hunk 3]
```diff
@@ -1,13 +1,43 @@
...
+
+
+## Introduction 
+
+### Butterfly hybrids
+It is known that a butterfly species may develop into multiple subspecies according to the habitat regions. The visual appearances (e.g., color patterns on the wings) of these subspecies can be drastically different.
+
+Normally, the same subspecies mate and produce children. Occasionally, different subspecies would mate and produce children that are considered **hybrids**, whose visual appearances are partially similar to each of their parents.
+
+In this challenge, children produced by the same-subspecies parents (i.e., **non-hybrids**) are treated as **normal** cases because they are much more frequently observed. In contrast, **hybrids** are treated as **anomaly** cases, not only because they are much less frequently observed--with some combinations not yet observed--but also because their visual appearances are much more variant and hardly predictive.
```
[the remark this answers is not in the document]
Comment (egrace479) [reply pages/overview.md:12]: Perhaps harder to predict? @pujols?

File: pages/overview.md [hunk 4]
```diff
@@ -1,13 +1,44 @@
...
+
+Normally, only individuals of the same subspecies mate and produce offspring. Occasionally, where the ranges of subspecies come into contact or overlap, individuals from different subspecies can mate, and produce offspring considered **hybrids**. The visual appearances of hybrids are _partially_ similar to _each_ of their parents.
+
+In this challenge, offspring produced by the same-subspecies parents (i.e., **non-hybrids**) are treated as **normal** cases because they are far more frequently observed. In contrast, **hybrids** are treated as **anomaly** cases, not only because they are much less frequently observed--with some combinations not yet observed--but also because their visual appearances are much more variant and hardly predictive.
+
+### Butterfly mimicry
+On a different dimension, two different butterfly species with overlapping geographic ranges may visually mimic each other. Such mimicry could help avoid shared predators, especially if one (or both) species is toxic or not palatable.
```
Comment (egrace479) [inline pages/overview.md:15]:
```suggestion
On a different dimension, two different butterfly species with overlapping geographic ranges may visually mimic each other. Such [mimicry](https://en.wikipedia.org/wiki/M%C3%BCllerian_mimicry) could help avoid shared predators, especially if one (or both) species is toxic or not palatable.
```

File: pages/evaluation.md [hunk 5]
```diff
@@ -1,35 +1,47 @@
...
+## The First Component of the Test Dataset
+This test data set comprises:
+- An image collection of butterfly **Species A subspecies**. These are non-hybrid instances.
+- An image collection of **Species A hybrids**, which are from parents from different subspecies of Species A. Note that in the training data, we only provide one type of hybrid (the "signal" hybrid), from one specific combination of parents. In the test data, there are also "non-signal" hybrids, which are from other combinations of parents. 
 
-Moreover, Species A has a mimic (or co-mimic) Species B. These two butterfly species have quite similar appearances to evade predators. The test data thus further contains:
-- A mimic butterfly species' (denoted as Species B) subspecies. Specifically, we only consider the two subspecies that correspond to the parent subspecies of Species A's signal hybrid.
-- Hybrids of these two subspecies.
-- The aim of this test set is to see whether an anomaly detection algorithm for butterflies is transferrable across mimic butterflies.
+The goal of the challenge is to distinguish between non-hybrid instances and hybrid instances.
```
Comment (egrace479) [inline pages/evaluation.md:9]:
```suggestion
The goal of the challenge is to elicit new ways of solving this problem through designing a model to distinguish between non-hybrid instances and hybrid instances.
```
Would this work? Though this might be more appropriate here as a task since it's just the first component part?

Comment (hlapp) [reply pages/evaluation.md:9]: I think this would work.

File: pages/overview.md [hunk 6]
```diff
@@ -1,13 +1,44 @@
...
+
+Besides the goal to develop an anomaly detection algorithm to distinguish between hybrids and non-hybrids for one species, this challenge aims to investigate whether such an algorithm is generalizable to the other visually mimicking species. 
+
+![subspecies_to_hybrids comparison different subspecies of Species A compared to two of its mimics](https://github.com/user-attachments/assets/8647e1f5-4f99-48c6-8325-fdfd0e5d4c21)
+
+
+## Setup Overview
+This challenge is designed to simulate a real-world biological scenario. Suppose a biologist studies a particular butterfly Species A. One day, the biologist finds that a subset of the images collected looks slightly abnormal in their visual appearance. After investigation, the biologist finds that these abnormal samples are hybrids produced by different subspecies of Species A, which are rarely observed. Since in theory, there are quadratically many possibilities of hybrids and the current collection only covers a small subset of them, the biologist seeks an anomaly detection algorithm to automatically identify (unseen) hybrid cases from future image collections of Species A.
```
Comment (egrace479) [inline pages/overview.md:23]:
```suggestion
This challenge is designed to simulate a real-world biological scenario. Suppose a biologist studies a particular butterfly Species A with many subspecies. One day, the biologist finds that a subset of the images collected looks slightly abnormal in their visual appearance. The biologist does not recognize the pattern as belonging to any of the subspecies on which their research is focused. After investigation, the biologist finds that these unusual samples are hybrids produced by different subspecies of Species A. Realizing that they may encounter other hybrids in future collections of images, the biologist seeks an anomaly detection algorithm to automatically identify (unseen) hybrid cases.
```
How does this sound?

Comment (hlapp) [reply pages/overview.md]: Sounds good, I applied it!

File: pages/evaluation.md [hunk 7]
```diff
@@ -1,35 +1,47 @@
...
-		1. Signal and non-signal hybrid subspecies of Species A. 
-		2. Signal hybrid subspecies of a mimic Species B.
-	* Participants may submit _one_ score on the "development" sets to be displayed on the leaderboard. This score can be removed and replaced with a newer or better score as they choose.
+	* The provided training data contains:
+		- Images of all Species A subspecies: these images are considered "normal" (not anomaly) cases.
+		- A signal set comprising the most common hybrid: these images are considered anomaly cases.*
+	* The goal is to develop an algorithm to detect hybrid instances (the anomaly cases).
+	* Upload your model: feedback will be provided on the development set until the end of the challenge:
+		1. Detect signal and non-signal hybrid subspecies of Species A. 
+		2. Detect signal hybrid subspecies of the mimic Species B.
```
Comment (egrace479) [inline pages/evaluation.md:29]:
```suggestion
		2. Detect hybrid subspecies among the mimic Species B (Species B subspecies are mimics of the Species A signal hybrid parent subspecies).
```
Attempt at clarifying. @pujols, @hlapp

Comment (hlapp) [reply pages/evaluation.md:29]: I would change _hybrid subspecies_ to _hybrids of subspecies_ or _subspecies hybrids_. (The subspecies themselves are normally not hybrids.)

**Review by hlapp — APPROVED** [review]
👍

File: pages/evaluation.md [hunk 8]
```diff
@@ -1,35 +1,47 @@
...
-	* The goal is to develop an algorithm to 1) detect hybrid butterflies of Species A's subspecies among their parent subspecies, and 2) detect the "signal" hybrid butterfly of a mimic Species (Species B) among its two parent subspecies.
-	* Upload your model; feedback will be provided on the development set for each component until the end of the challenge:
-		1. Signal and non-signal hybrid subspecies of Species A. 
-		2. Signal hybrid subspecies of a mimic Species B.
-	* Participants may submit _one_ score on the "development" sets to be displayed on the leaderboard. This score can be removed and replaced with a newer or better score as they choose.
+	* The provided training data contains:
+		- Images of all Species A subspecies: these images are considered "normal" (not anomaly) cases.
+		- A signal set comprising the most common hybrid: these images are considered anomaly cases.*
+	* The goal is to develop an algorithm to detect hybrid instances (the anomaly cases).
+	* Upload your model: feedback will be provided on the development set until the end of the challenge:
```
Comment (egrace479) [inline pages/evaluation.md:27]:
```suggestion
	* Upload your model: feedback will be provided on the development set until the end of the challenge; one submission is allowed per day.
```

File: pages/overview.md [hunk 9]
```diff
@@ -1,13 +1,44 @@
...
+### Test data
+We consider two sets of images in the test set:
+- One from Species A, comprising images from all the Species A subspecies, the signal hybrid, and the non-signal hybrids.
+- One from a mimicking Species B, comprising images from two subspecies of Species B and their hybrid. The two subspecies of Species B are the ones mimicking the parent subspecies of the signal hybrid of Species A.
 
 ## Timeline
 
-This ML Challenge starts on June 15, 2024 and will run through September [end day], 2024. Be sure to resubmit your preferred model from the development phase by [date, time]; it will then be run on the test sets to detect hybrids of Species A and its major hybrid mimics. Only one submission will be run against the test sets to determine your final score in the challenge.
-  
+This ML Challenge starts on September [start day], 2024, and will run through [end date]. Be sure to resubmit your preferred model from the development phase by [date, time]; it will then be run on the final test sets. Only one submission will be run against the test sets to determine your final score in the challenge.
```
Comment (egrace479) [inline pages/overview.md:38]:
```suggestion
This ML Challenge starts on September 11th, 2024, and will run through January 17th, 2025. Be sure to resubmit your preferred model from the development phase by January 17th at 11:59pm [AOE](https://www.timeanddate.com/time/zones/aoe); it will then be run on the final test sets. Only one submission will be run against the test sets to determine your final score in the challenge.
```
Update challenge deadlines.

File: pages/data.md [hunk 10]
```diff
@@ -1,12 +1,12 @@
 # Butterfly Hybrid Detection: Data
 
-This challenge uses data compiled from various Zenodo records of the Butterfly Genetics Group. [More Info Coming]
+This challenge uses data compiled from various Zenodo records of the Butterfly Genetics Group.
```
[the remark this answers is not in the document]
Comment (hlapp) [reply pages/data.md:3]: Perhaps we can link _Butterfly Genetics Group_ to something? (I tried searching, and a fitting URL didn't seem to recognizably surface; arguable another reason to provide a link?) And perhaps give an example for a pertinent Zenodo record, for who wants to get a quick idea?

[the remark this answers is not in the document]
Comment (egrace479) [reply pages/data.md:3]: I generally use [this link](https://zenodo.org/communities/butterfly/records?q=&f=subject%3ACambridge&l=list&p=1&s=10&sort=newest).

**Review by DavidCarlyn — APPROVED** [review]
